src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -552,6 +565,41 @@ class ProductaanvraagService @Inject constructor(
         assignZaak(createdZaak, zaakafhandelParameters)
         addInitiatorAndBetrokkenenToZaak(productaanvraag, createdZaak)
         cmmnService.startCase(createdZaak, zaaktype, zaakafhandelParameters, formulierData)
+
+        val to = extractEmail(extractIdentificatie(createdZaak))
+
+        val automaticEmailConfirmation = zaakafhandelParameters.automaticEmailConfirmation
+        if (automaticEmailConfirmation.enabled) {
+                val mailGegevens = MailGegevens(
+                    MailAdres(automaticEmailConfirmation.emailSender, null),
+                    MailAdres(to, null),
+                    MailAdres(automaticEmailConfirmation.emailReply, null),
+                    automaticEmailConfirmation.templateName.onderwerp,
+                    restMailGegevens.body,
+                    null,
+                    true
+                )
+                mailService.sendMail(mailGegevens, createdZaak.getBronnenFromZaak())
+                zaakService.setOntvangstbevestigingVerstuurdIfNotHeropend(createdZaak)
+        }
+    }
+
+    private fun extractIdentificatie(createdZaak: Zaak): String {
+        return when(val initiator = zaakService.getInitiatorFromZaak(createdZaak)) {
+            is MedewerkerIdentificatie -> { initiator.identificatie }
+            is NatuurlijkPersoonIdentificatie -> initiator.anpIdentificatie
+            is NietNatuurlijkPersoonIdentificatie -> initiator.annIdentificatie
```

Comment:
Niet Natuurlijk Persoon is much more complex! It can have different identificaties depending on which data is present. Please see my recent PR about this. Do you need to support NietNatuurlijkPersonen here at all? If so, please do not re-invent the wheel here but reuse existing code for retrieving the identificatie fields. See e.g. `RolNietNatuurlijkPersoon.getIdentificatienummer`